openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/WheelchairAttributesGraphStorage.java

```diff
@@ -69,20 +70,29 @@ public WheelchairAttributesGraphStorage()  {
 		trackTypeEncoder = new EncodedValueOld("tracktype", shift, 3, 1, 0, TRACK_TYPE_MAX_VALUE);
 		shift += trackTypeEncoder.getBits();
 
-		hasInclineEncoder = new EncodedValueOld("hasIncline", shift, 1, 1, 0, 1);
-		shift += 1;
 		inclineEncoder = new EncodedValueOld("incline", shift, 5, 1, 0, INCLINE_MAX_VALUE);
 		shift += inclineEncoder.getBits();
 
-		hasKerbHeightEncoder = new EncodedValueOld("hasKerbHeight", shift, 1, 1, 0, 1);
-		shift += 1;
 		kerbHeightEncoder = new EncodedValueOld("kerbHeight", shift, 4, 1, 0, KERB_MAX_VALUE);
 		shift += kerbHeightEncoder.getBits();
 
 		widthEncoder = new EncodedValueOld("width", shift, 5, 10, 0, WIDTH_MAX_VALUE);
 		shift += widthEncoder.getBits();
 
 		sideFlagEncoder = new EncodedValueOld("side", shift, 2, 1,0,2);
+		shift += sideFlagEncoder.getBits();
+
+		hasKerbHeightEncoder = new EncodedValueOld("hasKerbHeight", shift, 1, 1, 0, 1);
```

Comment:
EncodedValueOld should not be used in new code

Reply:
I was surprised to find the old flagencoder process in here anyway, why wasn't wheelchair refactored together with the other profiles? I also see lots of code duplication with respect to the other storage classes. 
The contributor just took over the pattern he found, and I think it exceeds the scope of this PR to address the refactoring of all the storage classes, which I think is required...

Reply:
Ok, I'll submit an issue for this.